src/node_worker.cc

```diff
@@ -150,7 +150,10 @@ class WorkerThreadData {
 
     isolate->Dispose();
 
-    // Need to run the loop one more time to close the platform's uv_async_t
+    // Need to run the loop twice more to close the platform's uv_async_t
+    // TODO(addaleax): It would be better for the platform itself to provide
+    // some kind of notification when it has fully cleaned up.
+    uv_run(&loop_, UV_RUN_ONCE);
```

Comment:
Should this be in a `#ifdef Win32` or similar?

Reply:
Why not UV_RUN_DEFAULT? My concern is that some platform-specific libuv change may at some point require a different arbitrary number or runs.

Reply:
@eugeneo Because we want to not keep spinning the event loop indefinitely, which could happen if e.g. addons still have active handles (which we currently crash for, which is what we want).

I agree that this is hacky – hence the `TODO` – but I wouldn’t expect libuv to change anything here (and if they do, we’d notice because they run Node.js in the libuv CI).

@Fishrock123 I don’t have strong feelings about that, I can add it if you want.

Reply:
Perhaps we should keep the behavior consistent then.